pynest/examples/wang_decision_making.py

```diff
@@ -0,0 +1,181 @@
+import nest
```

Comment:
Lacks copyright header and all documentation. See other examples for style for comments that is required for our conversion to notebooks.